Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -60,9 +60,30 @@ - (void)disable
     }
 }
 
-- (void)urlSessionTaskResume:(NSURLSessionTask *)sessionTask
+- (BOOL)addHeadersForRequestWithURL:(NSURL *)URL
```

Comment:
`l`: We only need `URL.absoluteString` so we could only pass the `absoluteString`.

Reply:
Passing in an actual `NSURL` rather than an `NSString` seems more clear and obvious to me, just by looking at the types you see what to expect.